pkgs/tools/networking/openconnect/default.nix

```
@@ -1,11 +1,34 @@
-{ stdenv, fetchurl, pkgconfig, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, fetchgit, darwin } :
+{ stdenv, fetchurl, fetchgit,
+  pkgconfig, makeWrapper,
+  openssl ? null, gnutls ? null,
+  gmp, libxml2, stoken, zlib,
+  nettools, gawk, openresolv, coreutils, gnugrep,
+  darwin } :
```

Please do not pass in darwin but the frameworks under darwin individually. 

```suggestion
  darwin
} :
```